release/models/bgp/openconfig-bgp-policy.yang

```diff
@@ -229,6 +236,19 @@ module openconfig-bgp-policy {
       the IGP cost using the enum (predefined value).";
   }
 
+  typedef bgp-as-mode-type {
```

Comment:
We use the term 'builtin' in other parts of OC to refer to special values or logic that is defined by the device. 

```suggestion
  typedef bgp-as-builtin-type {
```